bin/impl/install/accumulo.sh

```diff
@@ -59,6 +59,9 @@ else
   $SED "s#instance[.]zookeepers=localhost:2181#instance.zookeepers=$UNO_HOST:2181#" "$conf"/accumulo-client.properties
   $SED "s#auth[.]principal=#auth.principal=$ACCUMULO_USER#" "$conf"/accumulo-client.properties
   $SED "s#auth[.]token=#auth.token=$ACCUMULO_PASSWORD#" "$conf"/accumulo-client.properties
+  if [[ $ACCUMULO_VERSION =~ ^2\.0\.0.*$ ]]; then
```

Comment:
I wonder it 1.9 and 1.10 work.

Reply:
Probably... but I have not tested. It's probably best that, if you want to run Uno with one of those versions of Accumulo, you do so by changing the ZooKeeper version as well, to the version that Accumulo has been tested with (3.4.14 is probably best).